sdk-cpp/include/do_download.h

```diff
@@ -45,9 +46,23 @@ class download
     Attempting to set a download property on a version of windows earlier than 20H1 will not set the property and throw an exception with error code msdo::errc::do_e_unknown_property_id
     */
     void set_property(download_property key, const download_property_value& value);
-    download_property_value get_property(download_property key);
+    void get_property(download_property key, download_property_value& value);
```

Comment:
>void [](http://example.com/codeflow?start=4&length=4)

This should return the value instead, no?

Reply:
Yeah I'll have it as a return val instead of an out param here